students/bedkowska_julita/lesson_02_flow_control/bishop_move.py

```diff
@@ -0,0 +1,18 @@
+import math
+
+startX = int(input('Give the start column: '))
+startY = int(input('Give the start row: '))
+endX = int(input('Give the end column: '))
+endY = int(input('Give the end row: '))
+
+moveX = startX - endX
+moveY = startY - endY
+
+result = 'NO'
+if math.fabs(moveX) == math.fabs(moveY):
```

Comment:
Why math.fabs ?
We can use abs built in.
It's ok but for your information:
https://stackoverflow.com/questions/10772302/python-abs-vs-fabs